lldb/source/Plugins/Language/Swift/SwiftLanguage.cpp

```diff
@@ -1748,9 +1748,7 @@ bool SwiftLanguage::GetFunctionDisplayName(
     }
 
     if (inline_info) {
-      s << display_name;
-      s.PutCString(" [inlined] ");
-      display_name = inline_info->GetName().GetString();
+      s.PutCString("[inlined] ");
```

Comment:
Sorry for the back and forth, but now that we merged https://github.com/swiftlang/llvm-project/pull/10789 into 6.2, we can remove this

Reply:
Fixed!